### PR TITLE
Blog - Diluting DORA: How Marketers and Consultants Bastardize Engineering Best Practices

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
Preview URL: https://64f479fce822fe0d5078c604--ephemeral-narwhal-3ac3b2.netlify.app/blog/diluting-dora/

**Comment by thtmnisamnstr** [comment]
There are some formatting updates and a few small errors that I'll correct. Outside of that, looks good.

Is there any chance the hero image can be a bit more themed to match the images in the post? 80's professional wrestling would be the them I'm looking for, but that isn't a great description tbh. Here are some reference images that might convey what I'm looking for better than my words.

Wrestlemania 5 poster

Wrestlemania 6 poster

Big bright colors. Big block, metallic looking text. Lasers. Explosions. Basically, over the top. The most similar theme that is probably more available is The 80's.

Can you help out with this @msali123? If it's difficult, then we can use what we have, but, if you can make it more themed, I'd appreciate it.

**Comment by msali123** [comment]
@thtmnisamnstr sorry I thought I commented the preview link.

Okay i'll try to make something like this and share with you.

**Comment by adamgordonbell** [comment]
Some of the images seem fuzzy. I wonder if we can use an image upscaler and get better visuals? 

@msali123  could you try this, and see if they look better?
https://waifu2x.udp.jp/

**Comment by thtmnisamnstr** [comment]
I recommend using Upscayl for AI image enlargements, https://github.com/upscayl/upscayl. Solid, popular, and doesn't have any associations with manga erotica.

**Comment by msali123** [comment]
@thtmnisamnstr by hero image you mean the header right?
image

**Comment by thtmnisamnstr** [comment]
> @thtmnisamnstr by hero image you mean the header right? image

Yes.

**Comment by msali123** [comment]
header

@thtmnisamnstr please check if mountains look a bit over i can remove them.

**Comment by thtmnisamnstr** [comment]
@msali123 This looks good. It matches the examples really well. I have one request (it may not be possible). Could you put a vector image of a professional wrestling ring (a boxing ring would work too, they are the same)? Kinda like below but more fitting with our theme from a color and style perspective.
Boxing Ring vector example

**Comment by msali123** [comment]
header
@thtmnisamnstr lemme know if it doesnt looks good

**Comment by thtmnisamnstr** [comment]
That is amazing!

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+I have a somewhat unique and fairly thorough understanding of how marketers and consultants do this, how they take useful technical best practices and bastardize them for their own gain. I have a Computer Science degree and started my career as a computer programmer. I moved up into a system administrator role – the spiritual predecessor to DevOps roles. Eventually, I decided to get an MBA and worked for Deloitte as a senior consultant and eventually manager on tech projects afterward. After Deloitte, I moved into product marketing, working at DirecTV and then at several tech companies including New Relic, Twilio Segment, and, now, Earthly. I have been the user, the buyer, the marketer, and the consultant at different points in my career. I've seen this pattern repeated over and over.
+
+In this post, I'm going to focus specifically on DevOps Research and Assessment (DORA), what it is, why it matters, how marketers and consultants abuse it, and the negative impacts that abuse causes.
+Understanding DORA
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:23]: > "## Understanding DORA"

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 2]
```diff
@@ -61,7 +63,8 @@ I'm going to give marketers a lot of credit for popularizing useful technical be
...
 
 When I was the product marketer for the open source program at New Relic, I put together and the company executed a marketing program that sponsored the Oregon State University Open Source Lab, made a grant to freeCodeCamp for them to create a course on OpenTelemetry (that has been viewed 190k times, nice!), sponsored multiple tech podcasts, and started the company's GitHub Sponsors program that has now sponsored over 70 developers and organizations (do you have any idea how difficult it is to get a corporate procurement manager to give you approval to spend $5K on donations to individuals through one of GitHub's then-beta features?). Every item on that list is a good thing in my opinion. Every item on that list also works as marketing.
-Where Marketers Go Bad
+
+## Where Marketers Go Bad
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:67]: > ### Where Marketers Go Bad

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+When I was the product marketer for the open source program at New Relic, I put together and the company executed a marketing program that sponsored the Oregon State University Open Source Lab, made a grant to freeCodeCamp for them to create a course on OpenTelemetry (that has been viewed 190k times, nice!), sponsored multiple tech podcasts, and started the company's GitHub Sponsors program that has now sponsored over 70 developers and organizations (do you have any idea how difficult it is to get a corporate procurement manager to give you approval to spend $5K on donations to individuals through one of GitHub's then-beta features?). Every item on that list is a good thing in my opinion. Every item on that list also works as marketing.
+
+## Where Marketers Go Bad
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:67]: \> \#\#\# Where Marketers Go Bad

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+So here's the part where I tell you the pros of consultants. They sponsor a lot of the same technical reports, organizations, events, content creators, etc., as marketers. They also produce a lot of research-heavy reports that have useful info (unfortunately, they also produce a lot of sales pitches disguised as reports). Consultants probably do the best job of spreading technical best practices across industries, because they jump around industries so often. Sometimes you need a consultant because you need an outside perspective, you want advice from people with experience in areas your team is lacking, or you don't want to hire people full-time for temporary roles for work that is project-based or non-recurring. I can also say with certainty that my co-workers when I was a consultant were, for the most part, some of the most intelligent, well-rounded, well-composed people I've ever met in my life. If you need to put together an Academic Decathlon team or want to get the best tutors to get your child through high school and college with straight A's, consultants are the way to go.
+
+## Where Consultants Go Bad
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:90]: Change to: \#\#\# Where Consultants Go Bad

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
+---
+title: "Diluting DORA: How Marketers and Consultants Bastardize Engineering Best Practices"
+categories:
+  - Tutorials
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:4]: Change to:  - Articles

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+toc: true
+author: Gavin
+
+internal-links:
+ - diluting dora
+ - engineering best practices
+ - marketers and consultants bastardize engineering
+ - how to bastardize engineering best practices
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:12]: Change to: how engineering best practices are bastardized

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+excerpt: |
+    Learn how to write an engaging article with this checklist that covers everything from creating a catchy title to adding captivating images and incorporating internal links. Perfect for both new and experienced writers!
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:14]: Change to: Learn the ways marketers and consultants exploit DORA metrics. This blog post highlights ethical from unethical practices around the use of DORA metrics and offers guidance for tech experts. Learn how to protect the integrity of DORA metrics and make informed decisions.

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+title: "Diluting DORA: How Marketers and Consultants Bastardize Engineering Best Practices"
+categories:
+  - Tutorials
+toc: true
+author: Gavin
+
+internal-links:
+ - diluting dora
+ - engineering best practices
+ - marketers and consultants bastardize engineering
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:11]: Change to: marketers and consultants bastardize engineering best practices

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+This success and the trust that DORA has gained, especially the success of the DORA metrics, has made it a ripe vehicle to disguise selling snake oil and bullshit. The DORA metrics seem like a straightforward path to success, but, if you read the reports beyond just the metrics, it clearly isn't. The path to improving your company's DORA metrics was never prescriptive in any of the reports, and that's because it usually involves deep cultural change in an IT or Engineering organization. There is no cookie-cutter approach to organizational change. It's difficult and unique to each organization. Having simple, easy-to-measure metrics as goals and an ambiguous path to get there leaves a lot of room to create imaginary solutions and sell people things that may or may not help them.
+
+## How Marketers Abuse DORA
+
+### Not All Marketing is Bad
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:57]: Change to: \#\#\# Not All Marketing Is Bad

File: blog/_posts/2023-09-06-diluting-dora.md [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+
+For DORA, this exhibits itself by observability tool vendors creating solutions for things like DevOps or Cloud Adoption – things that require organizational change. They use DORA's State of DevOps Report and the DORA metrics to shape the steps in their solutions and the pitches they make to customers.
+
+The result is frequently a hefty amount of instrumentation that leaves the customer more locked into the vendor's tool (although this is getting less prevalent with OpenTelemetry), a bag of dashboards that show DORA metrics alongside a smattering of other metrics the vendor's solution deems important, and a fat bill.
+
+## How Consultants Abuse DORA
+### Consultants are Smart, and Sometimes You Need Them
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-06-diluting-dora.md:82]: Change to: \#\#\# Consultants Are Smart, and Sometimes You Need Them

**Comment by thtmnisamnstr** [comment]
@msali123 All my comments are in.

**Comment by msali123** [comment]
@thtmnisamnstr [Preview Link](https://64f96d4ff44bac3d5bcc5f7f--ephemeral-narwhal-3ac3b2.netlify.app/blog/diluting-dora/)

**Comment by thtmnisamnstr** [comment]
@msali123 The current upscaled images are pretty artifact-y. So I upscaled all of the images across several different upscaling models and picked out the best ones. They look better now. Can you update them? You can find each of them at:
* change-requests.jpg - https://drive.google.com/file/d/1bgEaeAWVfdXyB1gFslTluCSTVuPlLxbr/view?usp=drive_link
* megapowers.jpg - https://drive.google.com/file/d/1CFWYBm9tS9XELf-dsx-1y6VT59UykNFo/view?usp=drive_link
* cream-of-the-crop.jpg - https://drive.google.com/file/d/19nSVcnzOqgEz4iMcq4OxSUqpanBZoI4F/view?usp=drive_link
* deciphering.jpg - https://drive.google.com/file/d/1NkGdL4CdfmVanMC4tzovmUwbFglxtFJP/view?usp=drive_link
* marketer-good.jpg - https://drive.google.com/file/d/1Z78CJCaF0fxljImZyOjrF_KfYRtym7vu/view?usp=drive_link
* marketer-bad.jpg - https://drive.google.com/file/d/1PHJ5UN_lnQOYUJyHvGWf04viHbgyizUs/view?usp=drive_link

Also, I noticed that none on the in-article links are included. Can you add the links that are in the doc? https://docs.google.com/document/d/1MJekZe517PKLJ0QfeadUDwd3Y8kx2p3qL5FbjYWXmK8/edit?usp=sharing

**Comment by thtmnisamnstr** [comment]
@adamgordonbell When do you think we should publish this? I don't think we're holding it for anything. Sometime next week?

**Comment by adamgordonbell** [comment]
A couple days after Vlads post perhaps? Thursday?

**Comment by thtmnisamnstr** [comment]
@msali123 Can you also set this to publish next Thursday, Sept. 14?

**Comment by adamgordonbell** [comment]
@msali123  see slack messages about article to work on next.

**Comment by thtmnisamnstr** [comment]
Preview URL: https://650213ee6cfc3a42b0215205--ephemeral-narwhal-3ac3b2.netlify.app/blog/diluting-dora

File: blog/_posts/2023-09-14-diluting-dora.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+
+## Understanding DORA
+
+![IMegaPower]({{site.images}}{{page.slug}}/megapowers.jpg)\
+
+DORA started as a collaboration between a small group of researchers and [Puppet](https://www.puppet.com/) – one of the first DevOps tools on the market. They researched and gathered data from a wide swath of companies about their software development and delivery processes, producing the State of DevOps Report – one of the earliest and by far the most trusted reports on DevOps – from 2014 to 2017. The 2018 State of DevOps Report was noticeable sans Puppet but had a new diamond sponsor, Google. The company eventually acquired DORA in 2019, and they have continued to research and publish the State of DevOps Report every year since under the Google Cloud and DORA banners.
+
+> You can learn more about DORA's history in co-founder Jez Humble's 2019 blog post, ["DORA's Journey: An Exploration"](https://medium.com/@jezhumble/doras-journey-an-exploration-4c6bfc41e667).
+
+From the very earliest State of DevOps Reports, one of the primary takeaways is that companies with better IT performance are more productive, profitable, and have a higher market share.
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-14-diluting-dora.md:32]: Change "... are more productive, profitable, and have a higher..." -> "... are more productive, more profitable, and have a higher..."

File: blog/_posts/2023-09-14-diluting-dora.md [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+1. Deployment frequency
+2. Lead time for changes
+3. Mean time to recover from failure
+
+DORA has since expanded this to four metrics:
+
+1. Deployment frequency
+2. Lead time for changes
+3. Time to restore service
+4. Change failure rate.
```
Comment (thtmnisamnstr) [inline blog/_posts/2023-09-14-diluting-dora.md:47]: Remove the period, "4. Change failure rate." -> ""4. Change failure rate"

**Comment by thtmnisamnstr** [comment]
Left 2 comments with very small updates. LGTM after those are made.

Can we also update this: change "... He’s also a BJJ brown belt and..." -> "... He’s also a Brazilian jiu jitsu black belt and..."
image

**Comment by msali123** [comment]
Ok I'll change them

**Comment by thtmnisamnstr** [comment]
You don't need to. Adam merged it. I'll make the changes directly.

**Pull request merged**